src/lib/airspeed/airspeed.cpp

```diff
@@ -138,6 +138,31 @@ float calc_IAS_corrected(enum AIRSPEED_COMPENSATION_MODEL pmodel, enum AIRSPEED_
 				}
 				break;
 
+			case AIRSPEED_COMPENSATION_TFSLOT: {
```

Comment:
@roman-dvorak I think it would make sense to break this out into a separate PR.

Reply:
Okay, I'll create thes a PR as soon as possible.

Reply:
I created a separate PR (https://github.com/PX4/PX4-Autopilot/pull/18593) for these adjustments. Tomorrow I will test it with real hardware and, if necessary, mark it for merging. Thanks.